### PR TITLE
Implement new Channels and Filters for Targets

### LINKED ISSUE #134: Multiple slack endpoints
It would be great if the policy-reporter configuration would allow multiple slack targets.
Currently its only possible to setup one slack target.

**Desired behaviour**
- Define multiple slack targets
- Allow each slack target to have different channel
- Allow each slack target to have different priorities set

### PR DESCRIPTION
**Pull request by fjogeleit**
Signed-off-by: Frank Jogeleit <frank.jogeleit@web.de>

### CODE REVIEW

**Comment by matglas** [comment]
Nice work!

/lgtm

File: charts/policy-reporter/values.yaml [hunk 1]
```diff
@@ -161,6 +161,23 @@ target:
...
+#        exclude: ["debug", "info", "error"]
+    channels: []
+#    - host: "http://loki.loki-stack:3100"
+#      sources: []
+#      customLabels: {}
+#      filter:
+#        namespaces:
+#          include: ["develop"]
+#        priorities:
+#          exclude: ["debug", "info", "error"]
```
Comment (matglas) [inline charts/policy-reporter/values.yaml:180]: Would you want to add an example policies filter here too?

File: manifest/README.md [hunk 2]
```diff
@@ -108,6 +113,7 @@ s3:
   minimumPriority: "warning"
   skipExistingOnStartup: true
   sources: []
+  channels: []
 
 reportFilter:
   namespaces:
```
Comment (matglas) [inline manifest/README.md:119]: Maybe an example after this section would to know how to use the channels properties.

File: charts/policy-reporter/values.yaml (hunk 1 above)
Comment (fjogeleit) [reply charts/policy-reporter/values.yaml:180]: Thanks for the suggestion, I can also add a policy example at this point.

File: manifest/README.md (hunk 2 above)
Comment (fjogeleit) [reply manifest/README.md:119]: For a detailed explaination I will use the documentation.

**Pull request merged**